src/compas/geometry/_primitives/line.py

```diff
@@ -337,7 +338,32 @@ def transformed(self, T):
         line.transform(T)
         return line
 
+    def divide_by_count(self, number=10, include_ends=False):
+        """Return list of points from dividing the line by specific number of divisions
+
+        Parameters
+        ----------
+        number : integer
+            number of divisions
+        includeEnds : boolean
+            True if including start and end point in division points
+            False if not including start and end point in division points
+
+        Returns
+        -------
+        list of: compas.geometry.Point // Point as sequence of values xyz)
 
+        Example
+        --------
+        >>> line = Line([0.0,0.0,0.0],[5.0,0.0,0.0])
+        >>> line.divide_by_count(5, True)
+        [[0.0,0.0,0.0],[1.0,0.0,0.0],[2.0,0.0,0.0],[3.0,0.0,0.0],[4.0,0.0,0.0],[5.0,0.0,0.0]]
```

Comment:
I think the output is actually missing all spaces in between, so it will fail when it's executed by the testing thing.
```suggestion
        [[0.0, 0.0, 0.0], [1.0, 0.0, 0.0], [2.0, 0.0, 0.0], [3.0, 0.0, 0.0], [4.0, 0.0, 0.0], [5.0, 0.0, 0.0]]
```